### DISCUSSION TITLE
Can meson configure be used to disable the debug flag?

### DISCUSSION THREAD

**Discussion by akumbera** [mesonbuild/meson#12359] [Q&A] [answered]
I could not find a way in the documentation. Other options can be set, for example, by --optimization=0.
But, --debug=false or --debug=False or --debug false do nothing.

The only way I have found to set the debug flag to false is to configure the buildtype as plain.

**Comment by eli-schwartz** [comment, marked as answer]
I would suggest using the `-Ddebug=false` form, the `-D` option sets a builddir configuration option whereas `--opt` is a small syntactic sugar meant to resemble autotools `./configure` for builtin options only.

**Reply by akumbera** [reply]
Thank you very much for your help.
